cinema_service/urls.py

```diff
@@ -1,9 +1,14 @@
+from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
 
+from cinema_service import settings
+
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
 ]
+
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Make sure you don't push the media directory and images. This line serves static files from the MEDIA_URL, which should be used cautiously in production environments.